Apps/AccountDataAccess/test/unit/PatientRepositoryTests.cs

```diff
@@ -271,6 +271,44 @@ public async Task ShouldBlockAccess()
                     It.Is<AgentAudit>(aa => AssertAgentAudit(audit, aa))));
         }
 
+        /// <summary>
+        /// Can access data source.
+        /// </summary>
+        /// <param name="dataSource">The data source to check for access.</param>
+        /// <param name="canAccessDataSource">The value indicates whether the data source can be accessed or not.</param>
+        /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
+        [Theory]
+        [InlineData(DataSource.Note, true)]
+        [InlineData(DataSource.Medication, false)]
```

Comment:
Note is not blocked.  Note is not part of the data sources that are blocked.  So this tests if the data source can be accessed, which it can.  Medication is blocked.  So this data source cannot be accessed.